default/AI/ColonisationAI.py

```diff
@@ -1170,7 +1165,7 @@ def evaluate_planet(planet_id, mission_type, spec_name, empire, detail=None):
 
         detail.append("maxPop %.1f" % max_pop_size)
 
-        for special in ["MINERALS_SPECIAL", "CRYSTALS_SPECIAL", "METALOIDS_SPECIAL"]:
```

Comment:
The METALOID_SPECIAL wasn't actually removed from the game, it was simply (for some reason unknown to me) renamed to ELERIUM_SPECIAL.  Although none of the current species have the Mining Focus available, it is still in the content files and so this ref to metaloids should just be changed to ELERIUM_SPECIAL